src/components/Grid/Column.js

```diff
@@ -22,22 +34,20 @@ const Column = styled.div`
   ${props => getFlexProps(props.small)}
 
   ${mediumAndUp`
-    max-width: ${props => getSize(props.medium || props.small)}%;
     padding-right: ${spacing.gutters.mediumAndUp / 2}px;
     padding-left: ${spacing.gutters.mediumAndUp / 2}px;
-    ${props => getFlexProps(props.medium || props.small)}
+    ${({ medium, small }) =>
+      getColumnStylesMixin(getColumnValue(medium, small))}
```

Comment:
If neither medium nor small are defined, I think `max-width` will be set to NaN. Can you please safeguard against a bad return value from `getSize` when `val` is `undefined`?

Reply:
It will be fine as we have the default value of 12 for small